packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHomepageNavBar.tsx

```diff
@@ -67,7 +67,7 @@ export const SynapseHomepageNavBar: React.FunctionComponent<
         alignItems: 'center',
         columnGap: '5px',
         p: {
-          xs: '10px 0px',
+          xs: '10px 5px',
```

Comment:
Removed the padding that storybook adds when wrapping the component, and realized the synapse logo was bumping against the left side of the viewport.  Really only noticeable on mobile